automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/FireCloudClient.scala

```diff
@@ -54,11 +54,11 @@ trait FireCloudClient {
       case true =>
         val byteStringSink: Sink[ByteString, Future[ByteString]] = Sink.fold(ByteString("")) { (z, i) => z.concat(i) }
         val entityFuture = response.entity.dataBytes.runWith(byteStringSink)
-        Await.result(entityFuture, 100.millis).decodeString("UTF-8")
+        Await.result(entityFuture, 1.second).decodeString("UTF-8")
       case _ =>
         val byteStringSink: Sink[ByteString, Future[ByteString]] = Sink.fold(ByteString("")) { (z, i) => z.concat(i) }
         val entityFuture = response.entity.dataBytes.runWith(byteStringSink)
-        throw APIException(Await.result(entityFuture, 100.millis).decodeString("UTF-8"))
+        throw APIException(Await.result(entityFuture, 1.second).decodeString("UTF-8"))
```

Comment:
you could conceivably move these into a `val` somewhere to make it easy to change later ... but only 4 instances, possibly not worth it.